cards/cards.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../styles.css" />
+    <title>FoundOve UI</title>
+  </head>
+  <body>
+    <div class="container">
```

Comment:
Use more specific class name like card-container.

Reply:
Actually, I've not made any specific container for holding components in it. So, I've considered a generalized `container`  class which is coming from the root style.